wavelength-plugins/factions.js

```diff
@@ -49,47 +49,6 @@ function getFactionRank(user) {
 	}
 }
 
-// SHAMLESSLY RIPPED FROM ECONOMY.JS!
-
-function rankLadder(title, type, array, prop, group) {
```

Comment:
Don't forget exp uses it

Reply:
And ontime I've been meaning to make that a global tbh

Reply:
done